totalContributors.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import pathlib
+import os.path
+import sys
+import datetime
+
+def add_log(dict, input):
+    for dataset in input:
+        name = dataset.split("\t")[1]
+        value = int(dataset.split("\t")[0])
+        if name in dict:
+            dict[name] += value
+        else:
+            dict[name] = value
+
+def retrieve_log(repo, version):
+    return subprocess.run(['git', 'shortlog', '-s', version], capture_output = True, text = True, 
```

Comment:
The convention is to not have whitespaces around the `=`, when it is for keyword argument. You should run Black (https://github.com/psf/black) on your repo. It will auto-reformat your code.